djangocms_contentexpiry/models.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class ContentExpiry(models.Model):
+    activation_date = models.DateTimeField(auto_now_add=True)
+    expiry_date = models.DateTimeField(
+        default=datetime.now() + timedelta(days=30)
```

Comment:
This is a value that's calculated once on application startup. Need to make it a separate function or wrap it in `lambda: `. Also use `django.utils.timezone.now` instead.